### DISCUSSION TITLE
Displaying all Activities at once

### DISCUSSION THREAD

**Discussion by Noctis** [YetiForceCompany/YetiForceCRM#14583] [How to]
When I go into a Lead's details view and click on the "Activity" tab, I see a list of Calendar entries related to that Lead. There's a switch there which allows me to switch the list between "Current" and "History" events.

My question is: is there a way to display all the Calendar events at once there, both current & historic, like a third "All" option?

**Comment by rskrzypczak** [comment]
Yes, but only in the calendar record list :)
![image](https://user-images.githubusercontent.com/10221999/115723677-81937c80-a380-11eb-9d4c-484628ed7ad6.png)
